crates/bevy_pbr/src/render/shadows.wgsl

```diff
@@ -95,13 +97,9 @@ fn fetch_spot_shadow(light_id: u32, frag_position: vec4<f32>, surface_normal: ve
     // 0.1 must match POINT_LIGHT_NEAR_Z
     let depth = 0.1 / -projected_position.z;
 
-    #ifdef NO_ARRAY_TEXTURES_SUPPORT
-        return textureSampleCompare(view_bindings::directional_shadow_textures, view_bindings::directional_shadow_textures_sampler,
-            shadow_uv, depth);
-    #else
-        return textureSampleCompareLevel(view_bindings::directional_shadow_textures, view_bindings::directional_shadow_textures_sampler,
-            shadow_uv, i32(light_id) + view_bindings::lights.spot_light_shadowmap_offset, depth);
-    #endif
+	// Random number determined by trial and error that gave nice results.
+	let texel_size = 0.0134277345;
```

Comment:
The number isn't random, it's just determined by trial and error. And tabs -> 4 spaces.
```suggestion
     // Number determined by trial and error that gave nice results.
     let texel_size = 0.0134277345;
```